book/src/paper-wallet/usage.md

```diff
@@ -10,13 +10,92 @@ written to disk by unencrypted memory swaps. It is the user's responsibility to
 protect against this scenario.
 {% endhint %}
 
+## Creating a Paper Wallet
+
+Using the `solana-keygen` tool, it is possible to generate new seed phrases as
+well as derive a keypair from an existing seed phrase and (optional) passphrase.
+The seed phrase and passphrase can be used together as a paper wallet. As long
+as you keep your seed phrase and passphrase stored safely, you can use them to
+access your account.
+
+{% hint style="info" %}
+For more information about how seed phrases work, review this
+[Bitcoin Wiki page](https://en.bitcoin.it/wiki/Seed_phrase).
+{% endhint %}
+
+### Seed Phrase Generation
+
+Generating a new keypair can be done using the `solana-keygen new` command. The
+command will generate a random seed phrase, ask you to enter an optional
+passphrase, and then will display the derived public key and the generated seed
+phrase for your paper wallet.
+
+```bash
+solana-keygen new --no-outfile
+```
+
+{% hint style="warning" %}
+If the `--no-outfile` flag is **omitted**, the default behavior is to write the
+keypair to `~/.config/solana/id.json`
+{% endhint %}
+
+{% hint style="info" %}
+For added security, increase the seed phrase word count using the `--word-count`
+argument
+{% endhint %}
+
+For full usage details run:
+
+```bash
+solana-keygen new --help
+```
+
+### Public Key Derivation
+
+Public keys can be derived from a seed phrase and a passphrase if you choose to
+use one. This is useful for using an offline-generated seed phrase to
+derive a valid public key. The `solana-keygen pubkey` command will walk you
+through entering your seed phrase and a passphrase if you chose to use one.
+
+```bash
+solana-keygen pubkey ASK
+```
+
+{% hint style="info" %}
+Note that you could potentially use different passphrases for the same seed
+phrase. Each unique passphrase will yield a different keypair.
+{% endhint %}
+
+The `solana-keygen` tool assumes the use of the BIP39 standard English word
+list. If you chose to deviate from the word list or used a different language
```

Comment:
This is confusing since `solana-keygen` offers no mechanism to generate a seed phrase that *doesn't* use the BIP39 standard English word list.  How about something like:

The `solana-kegen` tool validates the seed phrase using the same BIP39 standard English word list it uses to generate seed phrases. If you generated your seed phrase with a different tool that uses a different word list, you can still use `solana-keygen`, but will need to add the `--skip-seed-phrase-validation` option.

Reply:
Sounds good!

@jstarry are you aware of whether we have similar language elsewhere that should also be tweaked?

Reply:
@t-nelson `grep` tells me just here and then we have `Skip validation of seed phrases. Use this if your phrase does not use the BIP39 official English word list` in the cli tool

Reply:
Thanks for `grep`ing for me!  I think the help message remains clear